pkg/services/ngalert/database_mig.go

```diff
@@ -107,3 +107,19 @@ func alertInstanceMigration(mg *migrator.Migrator) {
 	mg.AddMigration("add index in alert_instance table on def_org_id, def_uid and current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[0]))
 	mg.AddMigration("add index in alert_instance table on def_org_id, current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[1]))
 }
+
+func alertmanagerConfigurationMigration(mg *migrator.Migrator) {
+	alertConfiguration := migrator.Table{
+		Name: "alert_configuration",
+		Columns: []*migrator.Column{
+			{Name: "id", Type: migrator.DB_BigInt, IsPrimaryKey: true, IsAutoIncrement: true},
+			{Name: "alertmanager_configuration", Type: migrator.DB_Text, Nullable: false},
+			{Name: "alertmanager_templates", Type: migrator.DB_Text, Nullable: true},
+			{Name: "configuration_version", Type: migrator.DB_NVarchar, Length: 3, Nullable: true}, // In a format of vXX e.g. v1, v2, v10, etc
```

Comment:
I don't have the full picture but I just want to make sure that it's intentionally nullable.